src/annotator/config/index.js

```diff
@@ -31,6 +31,9 @@ export default function configFrom(window_) {
     }),
     query: settings.query,
     requestConfigFromFrame: settings.hostPageSetting('requestConfigFromFrame'),
+    configFrameAncestorLevel: settings.hostPageSetting(
```

Comment:
I think this change is left over from the first iteration of this PR and no longer applicable?